scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/service/ScheduleManager.java

```diff
@@ -710,4 +716,21 @@ private void deleteRecurringSchedule(RecurringScheduleEntity recurringScheduleEn
 		}
 	}
 
+    private void deleteActiveSchedules(String appId) {
+        try {
+            logger.info("Get all active schedules for application: " + appId);
+            List<ActiveScheduleEntity> activeSchedules = activeScheduleDao.findAllActiveSchedulesByAppId(appId);
```

Comment:
Yes could have simply added method to delete by appId , was trying to use existing method. Will change it.